snap/snapcraft.yaml

```diff
@@ -670,16 +687,20 @@ parts:
       # note - we specifically don't use arch
       case "$(dpkg --print-architecture)" in
         amd64)
-          FILE_NAME=kong-2.0.5.xenial.amd64.deb
-          FILE_HASH=450223a011dba0d6eeb6a192be368e7bcd45a5dc063c743ed6f3d37eb95237b8
+          # TODO: this is a temporary fix for the broken file sha from the bintray downloads
+          # there will be another PR to fix this and upgrade the version of Kong
+          FILE_NAME=kong-2.3.3.xenial.amd64.deb
+          FILE_HASH=0909891d483b0894460bb7f7b69381abaf6b4ebf2c7d13c7076a88087f739294
           ;;
         arm64)
           FILE_NAME=kong-2.0.5.xenial.arm64.deb
           FILE_HASH=15aa5fb2eb0c992afa277b09cb03b3b940df9704ab9e8d93327a72c0cb87eb09
           ;;
```

Comment:
Just to point out, arm64 hasn't been fixed.

Reply:
yea, my local machine is amd64 so I won;'t be able to touch it and test it here- will leave that to @siggiskulason's PR  on upgrading to Kong's version to fix that.

Reply:
Unfortunately, you won't be able to pass the pre-commit hook as it is.

Reply:
right- i'll rebase his PR once that is there.

Reply:
Rebased